src/types/package.rs

```diff
@@ -116,8 +116,8 @@ impl FromStr for PackageType {
             "npm" => Ok(Self::Npm),
             "python" | "pypi" => Ok(Self::PyPi),
             "maven" => Ok(Self::Maven),
-            "ruby" | "rubygems" => Ok(Self::RubyGems),
-            "nuget" => Ok(Self::Nuget),
+            "ruby" | "rubygems" | "gem" => Ok(Self::RubyGems),
+            "nuget" | "dotnet" => Ok(Self::Nuget),
```

Comment:
`nuget` is the correct `PURL` type. [Syft](https://github.com/anchore/syft), a popular sbom generation tool, uses `dotnet` for `nuget`.